.github/workflows/docs-link-check.yml

```diff
@@ -52,11 +52,22 @@ jobs:
         run: |
           cd docs
```

Comment:
While you're in here, can you remove this cd docs/ line? It was just added in a recent PR, but I want the check to run on all files, there are various READMEs outside docs folder